pyproject.toml

```diff
@@ -2,13 +2,110 @@
 requires = ["lsst-versions"]
```

Comment:
We had this pinned `lsst_versions > 1.3.0`in other repos  but that shouldn't be necessary